src/context/viewer-context.tsx

```diff
@@ -82,22 +82,23 @@ function useAuthedViewer() {
       }
     }
 
-    if (authToken) {
-      const sixtyDaysInSeconds = JSON.stringify(60 * 24 * 60 * 60)
-      auth
-        .handleCookieBasedAccessTokenAuthentication(
-          authToken,
-          sixtyDaysInSeconds,
-        )
+    const loadViewerFromToken = async () => {
```

Comment:
the similar methods here are async where their underlying actions are async so I made it async here as well